elyra/pipeline/airflow/provider_package_catalog_connector/airflow-provider-package-catalog.json

```diff
@@ -61,6 +61,25 @@
           "uihints": {
             "category": "Source"
           }
+        },
+        "auth_id": {
+          "title": "User Id",
+          "description": "User id that has read access for the specified URL resource",
+          "type": "string",
+          "uihints": {
+            "minLength": 1,
```

Comment:
Same comments as on the previous schema file relative to `minLength` location and the addition of a URI format specifier on the url property.